tests/test_connector.py

```diff
@@ -9,13 +9,15 @@
 import sys
 import uuid
 from collections import deque
+from concurrent.futures import ThreadPoolExecutor, as_completed
```

Comment:
I'd suggest import `futures` given that as_completed() also exists in asyncio and could cause confusion.